### PR TITLE
Add name to checkout analytics block

### PR DESCRIPTION
**Pull request by norgeindian**
## Description

With this adaption, it is possible to reference the block, so it can be adapted to custom needs.

### CODE REVIEW

**Review by cykolln — CHANGES_REQUESTED** [review]
@norgeindian thanks for contributing! Could you please add a line to the changelog under Unreleased describing this change? You can use our [changelogger](https://pypi.org/project/changelogged/) tool and run `changelogger add`. Then we can get this released with our next extension version.

**Comment by norgeindian** [comment]
@cykolln, sure, is it okay like this?

File: view/frontend/layout/checkout_index_index.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
         <referenceBlock name="content">
-            <block class="Klaviyo\Reclaim\Block\Initialize" template="Klaviyo_Reclaim::analytics/initialize.phtml" />
+            <block class="Klaviyo\Reclaim\Block\Initialize" template="Klaviyo_Reclaim::analytics/initialize.phtml" name="klaviyo.checkout.initialize" />
```
Comment (cykolln) [inline view/frontend/layout/checkout_index_index.xml:5]: sorry for the delay on this review - can you update this to `klaviyo_reclaim.checkout.initialize` just to follow our pattern for other block names

Comment (norgeindian) [reply view/frontend/layout/checkout_index_index.xml]: @cykolln , of course. Okay like that?

**Review by cykolln — APPROVED** [review]

**Comment by cykolln** [comment]
@norgeindian Thanks so much, we will get this out in the next release!

**Pull request merged**